### ISSUE TITLE
Add MathJax plug-in for MarkDocs

### ISSUE THREAD

**Issue by Nirus2000** [portfolio-performance/portfolio-help#111]
Add "[MathJax](https://www.mathjax.org/) and [KaTeX](https://katex.org/)" for displaying mathematical formulas and calculations.
https://squidfunk.github.io/mkdocs-material/reference/math/

**Comment by Sn1kk3r5** [comment]
Hi @Nirus2000,

sorry for the question, but where do you see the advantage compared to what we have today:

![{7A550967-B5D0-47DE-8E02-F2CA6F0856EA}](https://github.com/user-attachments/assets/0499cb87-18a8-4f52-9d30-eadc68fffd1e)

Cheers

**Comment by Nirus2000** [comment]
Ohh... i'm sorry 😀
